readthedocs/projects/models.py

```diff
@@ -1191,6 +1191,7 @@ class ImportedFile(models.Model):
     path = models.CharField(_('Path'), max_length=4096)
     md5 = models.CharField(_('MD5 checksum'), max_length=255)
     commit = models.CharField(_('Commit'), max_length=255)
+    build = models.IntegerField(_('Build id'), null=True)
```

Comment:
I put this as a integer field instead of a foreign key, to keep it simple, we don't require access to the object at all, just the unique number

Reply:
I wonder if it should be called `build_id` to clarify that.